clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/provider/GoogleInfrastructureProvider.groovy

```diff
@@ -51,15 +52,27 @@ class GoogleInfrastructureProvider extends AgentSchedulerAware implements Search
   ]
 
   final Map<String, SearchableProvider.SearchResultHydrator> searchResultHydrators = [
-      (INSTANCES.ns): new InstanceSearchResultHydrator(),
-      (BACKEND_SERVICES.ns): new BackendServiceResultHydrator()
+    (BACKEND_SERVICES.ns): new BackendServiceResultHydrator(),
+    (HTTP_HEALTH_CHECKS.ns): new HttpHealthCheckResultHydrator(),
+    (INSTANCES.ns): new InstanceSearchResultHydrator()
   ]
 
   @Override
   Map<String, String> parseKey(String key) {
     return Keys.parse(key)
   }
 
+  private static class HttpHealthCheckResultHydrator implements SearchableProvider.SearchResultHydrator {
+
+    @Override
+    Map<String, String> hydrateResult(Cache cacheView, Map<String, String> result, String id) {
+      CacheData healthCheckCacheData = cacheView.get(HTTP_HEALTH_CHECKS.ns, id)
+      return result + [
+        httpHealthCheck: JsonOutput.toJson(healthCheckCacheData.attributes.httpHealthCheck)
```

Comment:
Yes, the return type is `Map<String, String>`, so it's a serialized JSON string.